src/org/exist/start/start.config

```diff
@@ -62,7 +62,10 @@ lib/core/xmlrpc-server-%latest%.jar                always
 lib/core/clj-ds-%latest%.jar                       always
 lib/core/cglib-nodep-%latest%.jar                  always
 lib/core/j8fu-%latest%.jar			   always
+lib/core/icu4j-%latest%.jar                        always
```

Comment:
When testing this branch, the library was not found unless I used the explicit version '59_1' instead of %latest%. Maybe it fails because of the "_1". Not sure why I got it. I made a `clean-all` before testing.

Reply:
Is it possible you had an old version of icu4j in lib/user? In my testing I had encountered a `java.lang.NoSuchMethodError`, and when I asked Adam he said: "do you have an older icu jar kicking around in lib/user or somewhere?" I found I had icu4j-4_8_1_1.jar in lib/user (dated Aug 23, 2011!). Must've been there from a previous "download-icu4j" build step. Doing a `clean-all` didn't help. I had to manually delete the old jar file.

Reply:
@wolfgangmm Can you check what @joewiz  suggested please?

Reply:
No, I did not have an old icu. I was testing on a fresh checkout and it definitely fails to load `icu4j-59_1.jar`.

Reply:
@wolfgangmm Okay I see there was an issue when starting eXist from `org.exist.start.Main`, I have now fixed that whe